### PR TITLE
Haha & language grounding update

### PR DESCRIPTION
**Pull request by StephAO**

### CODE REVIEW

File: img/research/haha/haha_architecture.png
Comment (alecive) [inline img/research/haha/haha_architecture.png:1]: This file should still be transparent in the background. Please fix. Also, I would probably make it smaller? Maybe 2/3rd  smaller?

File: _posts/research/subteams/2021-03-16-algorithmic-social-hri.md [hunk 1]
```diff
@@ -26,25 +26,23 @@ We use an interdisciplinary approach, leveraging work from cognitive learning th
...
+To address this, we introduce HAHA: Hierarchical Ad Hoc Agents, a novel framework using hierarchical reinforcement learning to train an agent capable of navigating the intricacies of ad hoc teaming at a level of abstraction more akin to human collaboration.
+HAHA consists of a Worker and a Manager, which respectively focus on optimizing efficient sub-task completion and high-level team strategies.
+We evaluate HAHA in the Overcooked environment, demonstrating that it outperforms existing baselines in both quantitative and qualitative metrics, offering improved teamwork, better resilience to environmental shifts, and heightened agent intelligibility. 
+Furthermore, we show that the generalization ability of HAHA extends to changes in the environment and that our structure allows for the induction of new strategies not encountered during training.
+We posit that the advancements proposed in this paper form a crucial building block toward the realization of more secure and efficient human–AI teams.
```
Comment (alecive) [inline _posts/research/subteams/2021-03-16-algorithmic-social-hri.md:43]: `more secure and efficient` -> `safer and more efficient`?

File: _posts/research/subteams/2022-01-31-learning-modeling-robotics.md [hunk 2]
```diff
@@ -45,20 +45,27 @@ Moving forward, we are building hybrid models for manipulation in which physics-
...
-To this end, we aim at bridging the gap between the field of robotics and NLP to produce robots that can act and learn through language, and who in turn will generate experiences for it develop a richer understanding of language.
-
-<div class="row">
-    {% include post-sorter.html subtype="grounding_language_through_experience" %}
-</div>
+ - C. Paik, S. Aroca-Ouellette, A. Roncone, and K. Kann, _"The World of an Octopus: How Reporting Bias Influences a Language Model’s Perception of Color"_, 2021, In Empirical Methods in Natural Language Processing: EMNLP2021, [[PDF]]({{ site.url }}/papers/2021_Paik_EMNLP.pdf) [[BIB]]({{ site.url }}/papers/2021_Paik_EMNLP.bib)
+
+Natural language serves as the fundamental medium through which humans collaborate, providing a versatile framework for articulating tasks, exchanging information, and conveying intentions. 
+While modern language models have shown impressive results, it is essential to recognize a key distinction between how humans and language models acquire language.
+While humans learn language by experiencing it in the real world, present language models predominantly rely on text data alone for their learning.
```
Comment (alecive) [inline _posts/research/subteams/2022-01-31-learning-modeling-robotics.md:58]: "while..while" repetition

File: _posts/research/subteams/2022-01-31-learning-modeling-robotics.md [hunk 3]
```diff
@@ -45,20 +45,27 @@ Moving forward, we are building hybrid models for manipulation in which physics-
...
+To this end, we posit that the pursuit of text-only training is at a minimum inefficient, and possibly insufficient to achieve a human-like understanding of language.
+Our initial work PROST highlighted one area where text-only models fall short --- understanding questions related to physical reasoning --- and demonstrated that scale alone would be unlikely to solve this shortcoming.
```
Comment (alecive) [inline _posts/research/subteams/2022-01-31-learning-modeling-robotics.md:60]: not sure if the em-dashes are actually rendered in jekyll. Also, there should be no spaces in teh em-dashes

**Review by alecive — CHANGES_REQUESTED** [review]

File: _posts/research/subteams/2022-01-31-learning-modeling-robotics.md (hunk 3 above)
Comment (StephAO) [reply _posts/research/subteams/2022-01-31-learning-modeling-robotics.md]: Jekyll does render em-dashes, I checked, I've addressed the other comments

**Review by alecive — APPROVED** [review]

**Comment by alecive** [comment]
Ref #200

**Pull request merged**